components/31-molecules/contact-details/_contact-details.scss

```diff
@@ -32,6 +32,13 @@ $avatar-width: 14rem;
         box-shadow: none;
       }
     }
+
+    .accolade {
+      &::before,
+      &::after {
+        @include theme('background', 'color-secondary', 'highlight-background');
```

Comment:
```suggestion
        @include theme('background', 'color-secondary', 'highlight-inverted-background');
```

Reply:
Still need to commit this suggestion (if you agree of course).
You don't want 'highilight-background' --> that would be a lighter shade of color primary

You want 'contact-details-inverted-background-color' --> white.

You can even use the same themify function on line 27 to make sure they are both the same color.